package.json

```diff
@@ -8,16 +8,17 @@
     "lib"
   ],
   "scripts": {
+    "ci": "npm test -- --coverage --verbose && npm run integration",
     "build": "babel src -d lib",
     "dev": "babel -w src -d lib",
-    "lint": "eslint . --ignore-path .gitignore",
     "fix": "npm run lint -- --fix",
-    "preintegration": "npm pack",
     "integration": "jest --config test/config/integration.json",
+    "lint": "eslint . --ignore-path .gitignore",
+    "preintegration": "npm pack",
```

Comment:
I'd prefer the "pre"-lifecycles be defined before their corresponding "base" lifecycle. All the other changes in this file are fine, just move `preintegration` back above `integration`, please.